pkg/scannerv4/mappers/mappers.go

```diff
@@ -372,7 +359,7 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 		if vulnerabilities == nil {
```

Comment:
nit: this should be moved closer to when the map is used. I think right above `vulnerabilities[k] = v4Vuln`